website/static/js/fangorn.js

```diff
@@ -38,6 +38,23 @@ $.extend(EXTENSION_MAP, {
     gsheet: 'xlsx'
 });
 
+var _defaultIconState = function () {
+    return {
+        mode : 'bar',
+        generalIcons : {
+            search : { on : true, template : searchIcon },
```

Comment:
It's unclear where the defs come from. Would be good to have an explicit ```require``` or aliasing at the top of the file.